src/com/carolinarollergirls/scoreboard/core/impl/ScoreBoardImpl.java

```diff
@@ -152,14 +157,15 @@ public void reset() {
             removeAll(Period.NChild.JAM);
             removeAll(NChild.PERIOD);
             set(Value.CURRENT_PERIOD, getOrCreate(NChild.PERIOD, "0"));
+            noTimeoutDummy = new TimeoutImpl(getCurrentPeriod(), "noTimeout");
+            getCurrentPeriod().add(Period.Child.TIMEOUT, noTimeoutDummy);
+            set(Value.CURRENT_TIMEOUT, noTimeoutDummy);
```

Comment:
I'd expect this to be null, not a non-singleton placeholder.

Reply:
'null' would require special case handling for every acess to CURRENT_TIMEOUT.
And it's as singleton as Period 0 and Jam 0, which are used analoguously.